### DISCUSSION TITLE
How change rainbow state?

### DISCUSSION THREAD

**Discussion by ainou** [mechatroner/vscode_rainbow_csv#88] [Q&A] [answered]
I've somehow turned off rainbow effect (see image below) and can't turn it on again in Visual Studio Code. I saw how to change in Sublime but can't do it with VSC.

![Screenshot 2021-04-14 at 12 58 15](https://user-images.githubusercontent.com/873510/114749681-1677f380-9d4b-11eb-98a3-af9fc1e9640d.png)

**Comment by mechatroner** [comment, marked as answer]
Hi @ainou, there are 2 ways to turn it back ON: https://github.com/mechatroner/vscode_rainbow_csv/issues/49
Hope it helps!

**Reply by ainou** [reply]
Yes indeed, thanks!
